src/lib/rest/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ SET (SOURCES
     orionReply.cpp
     OrionError.cpp
     HttpStatusCode.cpp
+    curl_context.cpp
```

Comment:
And there is no class CurlContext in the file ... a plain struct and separate functions, but I don't think it is important, I'll change it to your style